samples/git-clone-sample

```diff
@@ -0,0 +1,36 @@
+kind: DevWorkspace
```

Comment:
This file is missing the `.yaml` extension.

Reply:
Woopsie :)